src/mui/button/SaveButton.js

```diff
@@ -6,20 +6,23 @@ import ContentSave from 'material-ui/svg-icons/content/save';
 import CircularProgress from 'material-ui/CircularProgress';
 import translate from '../../i18n/translate';
 
-class SaveButton extends Component {
+export class SaveButton extends Component {
 
     handleClick = (e) => {
         if (this.props.saving) {
             // prevent double submission
             e.preventDefault();
+        } else if (!this.props.submitOnEnter && this.props.handleSubmit) {
+            this.props.handleSubmit();
```

Comment:
`record-form-[index]` doesn't exist anymore (see #533), so you can safely use `record-form` everywhere (valid for both Create and Edit forms).